src/core/utils/options.ts

```diff
@@ -84,3 +84,25 @@ export function isConfigFileOption(option: keyof SWACLIConfig): boolean {
 export function isUserOrConfigOption(option: keyof SWACLIConfig): boolean {
   return isUserOption(option) || isConfigFileOption(option);
 }
+
+export function getFlagsUsed(options: SWACLIConfig) {
```

Comment:
Can we add at unit test for this function?

Reply:
unit tests in next iteration